x-pack/plugins/cases/public/components/create/form.tsx

```diff
@@ -70,6 +73,10 @@ export const CreateCaseFormFields: React.FC<CreateCaseFormFieldsProps> = React.m
     const { isSubmitting } = useFormContext();
     const { isSyncAlertsEnabled } = useCasesFeatures();
 
+    const { owner } = useCasesContext();
+    const availableOwners = useAvailableCasesOwners();
+    const canShowCaseSolutionSelection = !owner.length && availableOwners.length;
```

Comment:
do we still need to show the selector if we have only one availableOwner?

Reply:
Hey yeah, So they are aware that the other option could exist. Was a request from product, kind of to prod them to check out other solution, or request permission if they need it.